lib/ssl/doc/src/ssl.xml

```diff
@@ -1054,8 +1054,15 @@ fun(srp, Username :: binary(), UserState :: term()) ->
       <name name="client_cafile"/>
       <desc>
 	<p>Path to a file containing PEM-encoded CA certificates. The CA
-      certificates are used during server authentication and when building the
-      client certificate chain.</p>
+        certificates are used during server authentication and when building the
+        client certificate chain.</p>
+        <note><p>When PEM caching is enabled, files provided with
+        this option will be checked for updates at fixed time intervals specified by the
+        <seeapp marker="ssl:ssl_app#configuration">ssl_pem_cache_clean</seeapp> environment parameter.</p>
+        </note>
+        <note><p>The built-in PEM caching mechanism can be replaced with the
```

Comment:
This reads really strange to me. If we directly provide the file content as a binary instead of embedding it in a PEM file there is nothing to be cached. So the note I think should reflect this.